src/interface/swagger.json

```diff
@@ -950,9 +950,33 @@
         "AuthenticationValue": {
             "title": "AuthenticationValue",
             "description": "Contains the authentication value. The format depends on the authentication type used in the AuthenticationInfo complex type.",
-            "pattern": "^\\d{3,10}$|^\\S{1,64}$",
             "type": "string"
```

Comment:
unfortunately swagger 2.0 doesn't support oneOf, so auto mocking will not work for U2F case

Reply:
@eoln are you not editing the https://github.com/mojaloop/transaction-requests-service/blob/pisp/master/src/interface/TransactionRequestsService-swagger.yaml also?

Reply:
@rmothilal I edited it: https://github.com/mojaloop/transaction-requests-service/pull/47/files#diff-d5a56fe5aa69dae0c6905cb159ee9b4a